.github/workflows/_build_base.yaml

```diff
@@ -92,6 +92,15 @@ jobs:
           labels:
             org.opencontainers.image.created=${{ inputs.BUILD_DATE }}
 
+      - name: Update manifest and patches in-place - show diff
+        working-directory: .github/container
+        shell: bash -x -e {0}
+        run: |
+          build_event="${{ github.event_name }}"
```

Comment:
For better readability, I recommend the pattern where a dedicated predicate step would run the logic to determine whether version bump should happen, and then store the outcome as its step output. The actual bump step will then be conditional on the output of the predicate step. Take for example how the `PUBLISH` output is set in the `metadata` step of many nightly workflows.

Reply:
This way, it will also be easier to visually examine if the bump step did run in the GitHub Web GUI.

Reply:
Done